src/components/Chip/Chip.js

```diff
@@ -3,13 +3,15 @@ import PropTypes from "prop-types";
 
 import "./Chip.scss";
 
-const Chip = ({ value, lead = "", onDismiss }) => {
+const Chip = ({ value, lead = "", onClick, onDismiss, selected }) => {
   return (
-    <div className="p-chip">
-      {lead ? `${lead.toUpperCase()}: ` : null}
-      {value}
+    <div className="p-chip" data-selected={selected}>
```

Comment:
I would suggest we use `aria-pressed` instead of a data attr. https://gomakethings.com/button-state-and-accessibility/

Reply:
But maybe that should not be used on a div in this case.

Reply:
This is cool.. didn't know about it. It can also be used on a div by also adding `role="button"`